watertap/unit_models/aeration_tank.py

```diff
@@ -97,16 +97,7 @@ def variable_scaling_routine(
         )
         self.scale_variable_by_default(model.KLa, overwrite=overwrite)
         if model.config.has_aeration:
-            if "S_O" and "S_O2" in model.config.property_package.component_list:
-                self.scale_variable_by_default(
-                    model.control_volume.mass_transfer_term[0, "Liq", "S_O"],
-                    overwrite=overwrite,
-                )
-                self.scale_variable_by_default(
-                    model.control_volume.mass_transfer_term[0, "Liq", "S_O2"],
-                    overwrite=overwrite,
-                )
-            elif "S_O" in model.config.property_package.component_list:
+            if "S_O" in model.config.property_package.component_list:
```

Comment:
Why don't include S_O2 here? in modified ASM2d, our notation is S_O2

Reply:
No property packages have both S_O and S_O2, right? So I just check for them individually